Cargo.toml

```diff
@@ -103,8 +103,8 @@ zipf = "5.0.1"
 bitflags = "1.0.1"
 fail = "0.3"
 uuid = { version = "0.6", features = [ "serde", "v4" ] }
-grpcio = { version = "0.4", features = [ "secure" ] }
-raft = "0.4.3"
+grpcio = { version = "0.5.0-alpha.2", features = [ "secure" ] }
+raft = { git = "https://github.com/pingcap/raft-rs.git" }
```

Comment:
Can we specify a `rev` for these git deps? I think it is too easy to break otherwise.

Reply:
No, it won't. Version is locked in Cargo.lock. Besides, given a version requires kvproto also give a version, so upgrade raft means two repos both have to be updated. But I do think we should depend on a version once it's released, maybe 0.5.0-alpha.2.

Reply:
I think depending on the Cargo.lock file only is bad practice. It is a nice property that you can run `cargo update` and not have any breaking changes. By using bare git deps it means you have to be very careful about updating or you can get a breaking change.

> But I do think we should depend on a version once it's released, maybe 0.5.0-alpha.2.

Sure!